internal/db/repos.go

```diff
@@ -217,8 +218,11 @@ func (s *RepoStore) Count(ctx context.Context, opt ReposListOptions) (ct int, er
 const getRepoByQueryFmtstr = `
 SELECT %s
 FROM %%s
-WHERE deleted_at IS NULL
-AND %%s
+WHERE
+	deleted_at IS NULL
+AND %%s -- Populates "queryConds"
```

Comment:
Nitpick, but I think it's better to have these here, rather than in the method that populates the query:

```suggestion
AND (%%s) -- Populates "queryConds"
```

Reply:
Updated, thanks for the suggestion!